images/OWNERS

```diff
@@ -1,7 +1,6 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-- bentheelder
```

Comment:
I volunteer as tribute (I leave it to the other two approvers to decide if I should come in as a reviewer first)

Reply:
@ixdy is probably relatively done with images/, @krzyzacy WDYT?

Reply:
yep, thanks @spiffxp

Reply:
done